chart/values.yaml

```diff
@@ -914,10 +912,6 @@ config:
   elasticsearch:
     json_format: 'True'
     log_id_template: "{dag_id}_{task_id}_{execution_date}_{try_number}"
-  elasticsearch_configs:
-    max_retries: 3
-    timeout: 30
-    retry_timeout: 'True'
```

Comment:
These don't seem to be valid configs in any version? Do they ring a bell?

Reply:
This is valid: 

https://airflow.apache.org/docs/apache-airflow-providers-elasticsearch/stable/logging.html#writing-logs-to-elasticsearch-over-tls

This line: https://github.com/apache/airflow/blob/657384615fafc060f9e2ed925017306705770355/airflow/providers/elasticsearch/log/es_task_handler.py#L76 read all the configs passed to `elasticsearch_configs` and passes them as kwargs to ES client